File: Alkitab/src/main/java/yuku/alkitab/reminder/br/BootCompleteReceiver.java
package yuku.alkitab.reminder.br;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class BootCompleteReceiver extends BroadcastReceiver {
	@Override
	public void onReceive(Context context, Intent intent) {
		//App class will schedule the alarm. We don't need to put anything here since receiving will run the App
	}
}

File: Alkitab/src/main/java/yuku/alkitab/reminder/br/BootCompleteReceiver.kt
package yuku.alkitab.reminder.br

import android.annotation.SuppressLint
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class BootCompleteReceiver : BroadcastReceiver() {
    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    override fun onReceive(context: Context, intent: Intent) {
        //App class will schedule the alarm. We don't need to put anything here since receiving will run the App
    }
}
